src/AMOEBA/pair_amoeba.cpp

```diff
@@ -827,28 +827,35 @@ void PairAmoeba::init_style()
 
   // check if all custom atom arrays were set via fix property/atom
 
-  int flag,cols;
-
-  index_amtype = atom->find_custom("amtype",flag,cols);
-  if (index_amtype < 0 || flag || cols)
-    error->all(FLERR,"Pair {} amtype is not defined", mystyle);
-  index_amgroup = atom->find_custom("amgroup",flag,cols);
-  if (index_amgroup < 0 || flag || cols)
-    error->all(FLERR,"Pair {} amgroup is not defined", mystyle);
-
-  index_redID = atom->find_custom("redID",flag,cols);
-  if (index_redID < 0 || !flag || cols)
-    error->all(FLERR,"Pair {} redID is not defined", mystyle);
-  index_xyzaxis = atom->find_custom("xyzaxis",flag,cols);
-  if (index_xyzaxis < 0 || !flag || cols == 0)
-    error->all(FLERR,"Pair {} xyzaxis is not defined", mystyle);
-
-  index_polaxe = atom->find_custom("polaxe",flag,cols);
-  if (index_polaxe < 0 || flag || cols)
-    error->all(FLERR,"Pair {} polaxe is not defined", mystyle);
-  index_pval = atom->find_custom("pval",flag,cols);
-  if (index_pval < 0 || !flag || cols)
-    error->all(FLERR,"Pair {} pval is not defined", mystyle);
+  char const * names[6] = {"amtype", "amgroup", "redID",
+    "xyzaxis", "polaxe", "pval"};
+  int const flag_check[6] = {0, 0, 1, 1, 0, 1}; // correct type (0 int, 1 dbl)
+  int const cols_check[6] = {0, 0, 0, 3, 0, 0}; // xyzaxis 3 cols, all others 0
+  int const ghost_check[6] = {1, 1, 0, 0, 0, 1}; // which types need ghost; TO-DO: check
```

Comment:
Unless @srtee has a reason he thinks the ghost check for redID is not needed, I think this line should be {1,1,1,0,0,1} - which matches the settings in examples/amoeba/in.ubiquitin

Reply:
Happy to make the change -- I just haven't had time to do more testing myself this week.